src/permutation.jl

```diff
@@ -0,0 +1,59 @@
+export ExactPermutationTest, ApproximatePermutationTest
+
+function ptstats(x,y,f)
+    xy = vcat(x,y)
+    rx = 1:length(x)
+    ry = length(xy)-length(y)+1 : length(xy)
```

Comment:
Using parentheses instead of spacing to make precedence more obvious would be more standard IMHO.